gadget/gadget-amd64.yaml

```diff
@@ -25,12 +25,22 @@ volumes:
         type: C12A7328-F81F-11D2-BA4B-00A0C93EC93B
         size: 1200M
         update:
-          edition: 2
+          edition: 3
```

Comment:
Please do not update the edition yet - probably we do not want this change for units in the field, but only apply it to images built from scratch.